app/views/posts/show.html.erb

```diff
@@ -1,26 +1,32 @@
-<main>
-  <section class="card column">
-    <div class="card column">
-      <h2><%= @post.title %></h2>
-      <p><%= @post.text %></p>
-      <div class="right">
-        <span>Comments: <%= @post.comments_counter %></span>
-        <span>Likes: <%= @post.likes_counter %> </span>
-      </div>
-      <div class="button_field">
-        <%= link_to "Comment", new_user_post_comment_path(user_id: @post.author_id, post_id: @post.id ), class: "btn" %>
-        <%= form_with model: @like, url: user_post_likes_path(post_id: @post.id) do |form| %>
-          <%= form.submit "Like", class: "btn" %>
-        <% end %>
-      </div>
+<div class="container bg-dark text-white min-vh-100 pt-5 px-4 " >  
+
+  <div class="bg-primary px-3 py-3" >
+    <div class="d-flex align-items-center justify-content-between" >
+      <h2>Post #<%=@post.id%> by <%=@user.name%>  </h2>
```

Comment:
- [ ] There is no `@user` defined in your post show action but you are calling it here, leading to errors. Kindly fix the error. I think you meant this:

  ```
  @post.author.name
  ```

  ![image](https://github.com/Marcraphael12/rails-blog-app/assets/36542986/135d2673-9f20-422b-823d-f35adaae6b0e)